packages/web-react/src/components/Button/useButtonStyleProps.tsx

```diff
@@ -15,15 +15,21 @@ export interface ButtonStyles {
 }
 
 export function useButtonStyleProps<T extends ElementType = 'button'>(props: SpiritButtonProps<T>): ButtonStyles {
-  const { color, block, disabled } = props;
+  const { color, block, disabled, isSquare } = props;
```

Comment:
I don't like the inconsistency of `isSquare` and `block`, but it is your call.

Reply:
It will be solved in this sprint.